pagopa/io-app
# [IODPAY- 161] Navigate to initiative details from message CTA

## Short description
This PR enables navigation to an IDPay initiative details screen from message CTAs.

## List of changes proposed in this pull request
- Added linking options for `ioit://idapy/initative/:initiativeId"

## How to test
- Open a message with a CTA with the implemented route 
- Click on the CTA and see if the app navigates to the initiative details screen.
